cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -411,6 +422,23 @@ public ClusterOperationMetadata createCluster(String name, String imageVersion,
                              .setImageVersion(imageVersion)
                              .putAllProperties(dataprocProps));
 
+      //Add any Node Initialization action scripts
+      for (String action : conf.getInitActions()) {
+        builder.addInitializationActions(NodeInitializationAction.
+                newBuilder()
+                .setExecutableFile(action)
+                .build());
+      }
+
+      //Set Auto Scaling Policy
+      if (conf.isAutoPolicyEnabled()) {
+        builder.setAutoscalingConfig(AutoscalingConfig.
+                newBuilder()
```

Comment:
Same as above. When breaking a statement, the "." should be before the method call on the same line.

Reply:
Done